drafts/frozen-collections.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Blazor Project Structure
+domain: campzulu.hashnode.dev
+tags: .NET, C#, .NET 8
```

Comment:
I'm not sure whether Hashnode can handle unknown Tags or not, maybe we just need to try it out ^^

Reply:
Yes, changes should always be possible